app/assets/stylesheets/views/_transition-landing-page.scss

```diff
@@ -19,8 +22,123 @@ $red: #E61E32;
   color: govuk-colour("white");
 }
 
+.landing-page__header--with-bg-image {
+  padding-bottom: govuk-spacing(4);
+  padding-top: govuk-spacing(3);
+  background-color: $dark-blue;
+  border-top: govuk-spacing(2) solid $red;
+
+  @include govuk-media-query($from: $header-breakpoint) {
+    padding-bottom: govuk-spacing(4);
+    padding-top: govuk-spacing(8);
+    background-image: image-url('transition-2020-header-background.jpg');
+    background-size: cover;
+    background-position: top;
+    min-height: 500px;
+  }
+}
+
+.landing-page__header-text-block {
+  @include govuk-media-query($from: $header-breakpoint) {
+    padding-bottom: 80px;
+    padding-top: govuk-spacing(5);
+    background: linear-gradient(
+      to right, 
+      $dark-blue 0%, 
+      $dark-blue 66.66%, 
+      $transparent 66.66%, 
+      $transparent 100%
+    );
+  }
+}
+
+.landing-page__header-arrow-container {
+  text-align: right;
+
+  @include govuk-media-query($from: $header-breakpoint) {
+    position: absolute;
+    left: 52%;
+  }
+
+  @include govuk-media-query($from: 1000px) {
+    left: 53%;
+  }
+
+  @include govuk-media-query($from: 1100px) {
+    left: 54%;
+  }
+
+  @include govuk-media-query($from: 1200px) {
+    left: 56%;
+  }
+
+  @include govuk-media-query($from: 1300px) {
+    left: 57%;
+  }
+
+  @include govuk-media-query($from: 1400px) {
+    left: 58%;
+  }
+
+  @include govuk-media-query($from: 1800px) {
+    left: 59%;
+  }
+}
+
+.landing-page__header-arrow {
+  width: 90px;
+
+  @include govuk-media-query($from: $header-breakpoint) {
+    width: 165px;
+  }
+}
+
+.landing-page__take-action-button {
+  .govuk-button {
```

Comment:
I'm late to this, but I notice that this styles a component outside of the gem, which breaks our component isolation rule. Is there a plan to move this style to an option on the component itself?

Reply:
I don't think this is a style we want to bring into the component. This was done as an override for this specific button (to match the Brexit buttons).

This is experimental and we are looking at whether we want to go back to the original styling.

Reply:
Thanks. The style should either be removed or moved into the gem - we can't have applications styling components like this, it makes developing components harder and introduces more risk when doing so (for example if the default colour of button text was changed, it could introduce an colour contrast accessibility issue).